bin/resources/shaders/dx11/tfx.fx

```diff
@@ -1,5 +1,3 @@
-#ifdef SHADER_MODEL // make safe to include in resource file to enforce dependency
```

Comment:
Is it fine to remove the check?

Reply:
This no longer goes into a resource file (and hasn't for a year or so), so I'd guess so